client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -192,16 +190,7 @@ const siteSetupFlow: Flow = {
 			switch ( currentStep ) {
 				case 'options': {
 					if ( intent === 'sell' ) {
-						/**
```

Comment:
Might be good to leave this comment in for context.

Reply:
Agreed! :)